### PR TITLE
Method on NDCube for showing physical axes for each array axis

### PR DESCRIPTION
**Pull request by DanRyanIrish**
### Description

This PR creates a new method on ```NDCube``` that returns the physical types associated with each data/array axis.  For example, given a cube with 4 data axes and 4 world axis physical type, lon, lat, wavelength and time:
```python
>>> cube.array_axis_physical_types
[('custom:pos.helioprojective.lon', 'custom:pos.helioprojective.lat'),
('custom:pos.helioprojective.lon', 'custom:pos.helioprojective.lat'),
('em.wl',),
('time',)]
```

Fixes #<Issue Number>

### CODE REVIEW

**Comment by DanRyanIrish** [comment]
@Cadair: Any philosophical objections to this convenience method?

File: changelog/281.feature.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+New method on NDCube (array)axis_physical_types) to show which physical types are associated with each array axis.
```
Comment (DanRyanIrish) [inline changelog/281.feature.rst:1]:
```suggestion
New method on NDCube (array_axis_physical_types) to show which physical types are associated with each array axis.
```

File: ndcube/ndcube.py [hunk 2]
```diff
@@ -223,6 +223,13 @@ def world_axis_physical_types(self):
 
         return tuple(ctype[::-1])
 
+    @property
+    def array_axis_physical_types(self):
```
Comment (Cadair) [inline ndcube/ndcube.py:227]: needs a docstring

File: changelog/281.feature.rst [hunk 3]
```diff
@@ -0,0 +1 @@
+New method on NDCube (array_axis_physical_types) to show which physical types are associated with each array axis.
```
Comment (Cadair) [inline changelog/281.feature.rst:1]:
```suggestion
Added a new method `ndcube.NDCube.array_axis_physical_types` to show which physical types are associated with each array axis.
```

**Pull request merged**